examples/integration/CommandLine/CommandLineTool/BUILD

```diff
@@ -16,6 +18,23 @@ macos_command_line_application(
     deps = [":tool.library"],
 )
 
+apple_universal_binary(
+    name = "UniversalCommandLineTool",
+    binary = ":tool.binary",
+    forced_cpus = [
+        "x86_64",
+        "arm64",
+    ],
+    minimum_os_version = "12.0",
```

Comment:
Does using 11 here allow sharing of any dependencies? Or is the transition different so they just end up being different hashes then?

Reply:
12 wasn't an intentional diff from the original tool above. I'll change.